pkg/imgpkg/bundle/bundle_images_lock.go

```diff
@@ -18,6 +18,7 @@ import (
 )
 
 func (o *Bundle) AllImagesLock(concurrency int) (*ImagesLock, error) {
+	// TODO: See if we can add the locations here
```

Comment:
per our discussion, this method should probably start returning Bundles that were found. we can still keep returning ImagesLock but we should probably rename it to ImageRefs since that's what it is at this point.

Reply:
fixed